tower_cli/resources/project.py

```diff
@@ -67,6 +69,8 @@ def create(self, organization=None, monitor=False, timeout=None,
         This would be a shared class with user, but it needs the ability
         to monitor if the flag is set.
         """
+        if 'job_timeout' in kwargs and 'timeout' not in kwargs:
```

Comment:
Interesting fact here: the `if` clause here is solely to make unit tests happy, because our unit test code does not populate `kwargs` with the complete set of possible items, but in practice it does.

I don't think we need to refactor our unit test code base for this reason alone, but we might need to be careful in mimicking the real world when writing future test codes.